test/e2e/apimachinery/watch.go

```diff
@@ -325,7 +325,7 @@ var _ = SIGDescribe("Watchers", func() {
 
 	/*
 	   Testname: watch-consistency
-	   Release : v1.15
+	   Release: v1.15
 	   Description: Ensure that concurrent watches are consistent with each other by initiating an additional watch
```

Comment:
Thanks for the PR! Small nit: the release is out of order.

Reply:
Updated